critiquebrainz/frontend/templates/artist.html

```diff
@@ -159,6 +159,13 @@
       <div class="favicon-container"><img src="{{ url_for('static', filename='favicons/musicbrainz-16.ico') }}" /></div>
       <a href="//musicbrainz.org/artist/{{ artist.id }}"><em>{{ _('Edit on MusicBrainz') }}</em></a>
     </div>
+    <div id="cover-art-annotation">
+      <span class="text-muted">
+        Covers provided by :
```

Comment:
Better wording might be "Cover art provided by Cover Art Archive." No need to use colons then. CAA is our only provider.